src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/DefaultDocumentWriter.cs

```diff
@@ -99,7 +99,12 @@ public override void VisitDocument(DocumentIntermediateNode node)
                     }
                 }
 
+                Context.Writer.WriteLine("// <auto-generated/>");
+                Context.Writer.WriteLine("#pragma warning disable 1591");
```

Comment:
Chain writelines. Also, indent the Write's in the `IsNullOrEmpty` if above this.